next.config.js

```diff
@@ -30,18 +30,37 @@ const nextConfig = withCSS({
     // eslint-disable-next-line no-param-reassign
     config.node = { fs: 'empty' };
 
-    config.module.rules.push({
-      test: /\.svg$/,
-      use: [
-        {
-          loader: 'react-svg-loader',
-          options: {
-            svgo: svgoConfig,
+    config.module.rules.push(
+      {
+        test: /\.svg$/,
+        use: [
+          {
+            loader: 'react-svg-loader',
+            options: {
+              svgo: svgoConfig,
+            },
           },
-        },
-      ],
-    });
-
+        ],
+      },
+      {
```

Comment:
You know what...

Let's just make the names file an exported javascript object.

We can drop the new loaders, remove the `cDM` hook, and still maintain a separation of version control, while also keeping a very simple file to edit.